### DISCUSSION TITLE
Home Assistant Mode

### DISCUSSION THREAD

**Discussion by tunemanbbs** [mrlt8/docker-wyze-bridge#232] [General]
🚀 STARTING DOCKER-WYZE-BRIDGE v1.0.3
🏠 Home Assistant Mode
2021/11/14 13:55:30 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'user' data
2021/11/14 13:55:30 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'auth' data
2021/11/14 13:55:30 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2021/11/14 13:55:31 [WyzeBridge] 💾 Saving 'auth' to local cache...
2021/11/14 13:55:31 [WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
2021/11/14 13:55:32 [WyzeBridge] 💾 Saving 'user' to local cache...
2021/11/14 13:55:32 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'cameras' data
2021/11/14 13:55:32 [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
2021/11/14 13:55:33 [WyzeBridge] 💾 Saving 'cameras' to local cache...
🎬 STARTING ALL 11 CAMERAS
2021/11/14 13:55:33 [WyzeBridge] Starting rtsp-simple-server v0.17.8

After .7.6.0 the updates have been staying like this.   For some reason, it can't start the RTSP server if I roll back its fine.

**Comment by tunemanbbs** [comment]
21-11-14 13:51:14 INFO (SyncWorker_3) [supervisor.docker.addon] Starting Docker add-on mrlt8/wyze-bridge with version 1.0.3
21-11-14 13:52:06 INFO (SyncWorker_4) [supervisor.docker.interface] Stopping addon_7094bb28_docker-wyze-bridge application
21-11-14 13:52:07 INFO (SyncWorker_4) [supervisor.docker.interface] Cleaning addon_7094bb28_docker-wyze-bridge application
21-11-14 13:52:08 INFO (SyncWorker_7) [supervisor.docker.addon] Starting Docker add-on mrlt8/wyze-bridge with version 1.0.3
21-11-14 13:55:27 INFO (SyncWorker_1) [supervisor.docker.interface] Stopping addon_7094bb28_docker-wyze-bridge application
21-11-14 13:55:28 INFO (SyncWorker_1) [supervisor.docker.interface] Cleaning addon_7094bb28_docker-wyze-bridge application
21-11-14 13:55:30 INFO (SyncWorker_5) [supervisor.docker.addon] Starting Docker add-on mrlt8/wyze-bridge with version 1.0.3

**Comment by tunemanbbs** [comment]
<html>
<body>
<h1 class="card-header" style="color: rgb(33, 33, 33); font-family: Roboto, Noto, sans-serif; font-size: var(--ha-card-header-font-size, 24px); letter-spacing: -0.012em; line-height: 48px; padding: 12px 16px 16px; display: flex; margin-block: 0px; font-weight: normal; justify-content: space-between; align-items: center; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="card-header-text">System Health</div><ha-button-menu corner="BOTTOM_START" slot="toolbar-icon" style="display: inline-block; position: relative;"><div><slot name="trigger"><ha-icon-button slot="trigger" style="display: inline-block; outline: none;"><mwc-icon-button title="Copy menu" style="display: inline-block; outline: none; --mdc-ripple-color: currentcolor; -webkit-tap-highlight-color: transparent; vertical-align: top; --mdc-theme-on-primary: currentColor; --mdc-theme-text-disabled-on-light: var(--disabled-text-color);"><button class="mdc-icon-button mdc-icon-button--display-flex" aria-label="Copy menu" style="font-size: 24px; width: var(--mdc-icon-button-size, 48px); height: var(--mdc-icon-button-size, 48px); padding: calc( (var(--mdc-icon-button-size, 48px) - var(--mdc-icon-size, 24px)) / 2 ); display: inline-flex; position: relative; box-sizing: border-box; border: none; outline: none; background-color: transparent; fill: currentcolor; color: inherit; text-decoration: none; cursor: pointer; user-select: none; z-index: 0; overflow: visible; align-items: center; justify-content: center; vertical-align: top;"><span><slot><ha-svg-icon style="display: block; align-items: center; justify-content: center; position: relative; vertical-align: middle; fill: currentcolor; width: var(--mdc-icon-size, 24px); height: var(--mdc-icon-size, 24px);"><svg preserveAspectRatio="xMidYMid meet" focusable="false" viewBox="0 0 24 24"><g><path d="M19,21H8V7H19M19,5H8A2,2 0 0,0 6,7V21A2,2 0 0,0 8,23H19A2,2 0 0,0 21,21V7A2,2 0 0,0 19,5M16,1H4A2,2 0 0,0 2,3V17H4V3H16V1Z"></path></g></svg></ha-svg-icon></slot></span></button></mwc-icon-button></ha-icon-button></slot></div><mwc-menu></mwc-menu></ha-button-menu></h1><div class="card-content" style="padding: 0px 16px 16px; margin-top: -8px; color: rgb(33, 33, 33); font-family: Roboto, Noto, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">

Version | core-2021.11.3
-- | --
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.7
Operating System Family | Linux
Operating System Version | 5.10.75
CPU Architecture | x86_64
Timezone | America/Chicago

</div>
</body>
</html>

**Comment by tunemanbbs** [comment]
<html>
<body>
<div class="card-header" style="justify-content: space-between; display: flex; align-items: center; color: rgb(33, 33, 33); font-family: Roboto, Noto, sans-serif; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><h3>Home Assistant Supervisor</h3><a class="manage" href="http://192.168.2.6:8123/hassio" style="color: var(--primary-color); text-decoration: none;"><mwc-button style="display: inline-flex; outline: none; -webkit-tap-highlight-color: transparent; vertical-align: top;"><button id="button" class="mdc-button   " aria-label="" style="-webkit-font-smoothing: antialiased; font-family: var(--mdc-typography-button-font-family, var(--mdc-typography-font-family, Roboto, sans-serif)); font-size: var(--mdc-typography-button-font-size, 0.875rem); line-height: inherit; font-weight: var(--mdc-typography-button-font-weight, 500); letter-spacing: var(--mdc-typography-button-letter-spacing, 0.0892857em); text-decoration: var(--mdc-typography-button-text-decoration, none); text-transform: var(--mdc-typography-button-text-transform, uppercase); position: relative; display: inline-flex; align-items: center; justify-content: center; box-sizing: border-box; min-width: 64px; border: none; outline: none; user-select: none; appearance: none; overflow: hidden; vertical-align: middle; background: transparent; padding-top: 0px; padding-right: var(--mdc-button-horizontal-padding, 8px); padding-bottom: 0px; padding-left: var(--mdc-button-horizontal-padding, 8px); height: 36px; border-radius: var(--mdc-shape-small, 4px); flex: 1 1 auto; color: var(--mdc-theme-primary, #6200ee);"><span class="leading-icon"><slot name="icon"></slot></span><span class="mdc-button__label" style="position: relative;"></span><span class="slot-container   " style="display: inline-flex; align-items: center; justify-content: center;"><slot>MANAGE</slot></span><span class="trailing-icon"><slot name="trailingIcon"></slot></span></button></mwc-button></a></div>

Host Operating System | Home Assistant OS 6.6
-- | --
Update Channel | stable
Supervisor Version | supervisor-2021.10.8
Docker Version | 20.10.8
Disk Total | 30.8 GB
Disk Used | 5.5 GB
Healthy | true
Supported | true
Board | ova
Supervisor API | ok
Version API | ok
Installed Add-ons | Docker Wyze Bridge (1.0.3)


</body>
</html>

**Comment by tunemanbbs** [comment]
FYI here is the log from the working version that I just rolled back to again.

🚀 STARTING DOCKER-WYZE-BRIDGE v0.7.6
🏠 Home Assistant Mode
2021/11/14 14:08:13 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'user' data
2021/11/14 14:08:13 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'auth' data
2021/11/14 14:08:13 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2021/11/14 14:08:16 [WyzeBridge] 💾 Saving 'auth' to local cache...
2021/11/14 14:08:16 [WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
2021/11/14 14:08:16 [WyzeBridge] 💾 Saving 'user' to local cache...
2021/11/14 14:08:16 [WyzeBridge] ♻️ FORCED REFRESH - Removing local 'cameras' data
2021/11/14 14:08:16 [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
2021/11/14 14:08:17 [WyzeBridge] 💾 Saving 'cameras' to local cache...
🎬 STARTING ALL 11 CAMERAS
2021/11/14 14:08:18 [WyzeBridge] Starting rtsp-simple-server v0.17.3
2021/11/14 14:08:20 [Logan] 🎉 Starting HD 120kb/s Stream for WyzeCam V3 in "LAN mode" FW: 4.36.3.19 IP: 192.168.2.167 WiFi: 97%
2021/11/14 14:08:21 [Barn1] 🎉 Starting HD 120kb/s Stream for WyzeCam V3 in "LAN mode" FW: 4.36.3.19 IP: 192.168.2.221 WiFi: 99%
2021/11/14 14:08:21 [Weather] 🎉 Starting HD 120kb/s Stream for WyzeCam V2 in "LAN mode" FW: 4.28.4.49 IP: 192.168.2.140 WiFi: 100%

**Reply by unrly** [reply]
Not to hijack, but how did you roll back to 0.7.6 in HA?

**Reply by tunemanbbs** [reply]
there is an option that when you upgrade it does a backup.  "I restored the backup"

**Reply by unrly** [reply]
@tunemanbbs Thanks, that's what I thought since I couldn't find anything online that suggested it was possible via command line or the UI. I don't have a backup of the older version, but 0.7.6 is the last good version that actually worked for me.

**Comment by mrlt8** [comment]
hmm, we did remove Apparmor in v1.0.1, but I don't think that is causing your issue, since it looks like your add-on can fetch data from the api.
Does restarting the add-on a couple of times help?

**Reply by tunemanbbs** [reply]
Not at all.  it will always just sit there.  I mean I can try disabling AppArmor but that just restarts it if it stops.

**Reply by tunemanbbs** [reply]
Any idea how to capture additional logging?

**Reply by tunemanbbs** [reply]
Just for reference.  This is what im doing with the app.  I don't think it matters. ...   https://www.youtube.com/watch?v=m8mBaA-KScc

**Reply by mrlt8** [reply]
I think this is probably also related to #211. 
For some reason, the bridge will sometimes crash when trying to connect to the camera which causes all the streams to stop. In your case, the bridge seems to get stuck on the initial connect so the streams never go up.

**Reply by tunemanbbs** [reply]
Should i throw debug on?  because clearly, this is a change between the 076 and 1.0x
